### DISCUSSION TITLE
taro3.x 使用cli新建项目时没有选择使用ts，现在我如何在项目中使用ts

### DISCUSSION THREAD

**Discussion by GoldWood2000** [NervJS/taro#13522] [Q & A] [answered]
### 这个特性解决了什么问题？
项目中可以保证js和ts共存嘛？例如新的模块使用ts开发，原有模块保留，因为我不是很懂工程化这块。

### 这个 API 长什么样？
项目中可以保证js和ts兼容使用

**Comment by ZakaryCode** [comment, marked as answer]
可以直接参考 ts 项目模板修改项目，js/ts 共存只需要确保 tsconfig 中 `allowJs: true` 即可
